src/platforms/common/configuration/options.mdx

```diff
@@ -281,7 +281,31 @@ The number of context lines for each frame when loading a file.
 
 <ConfigKey name="initial-scope" supported={["javascript","node"]}>
 
-Data to be set to the initial scope.
+Data to be set to the initial scope. Initial scope can be defined either as an object as follows
```

Comment:
This construction threw me off for a second because I hadn't read further down the page yet to find the "or" so I just assumed it was missing in the first statement. Suggest making this a bit more foolproof.

```suggestion
Data to be set to the initial scope. Initial scope can be defined either as an object or a callback function, as shown below.

Object:
```